.circleci/config.yml

```diff
@@ -511,7 +640,7 @@ commands:
             # Make this environment variable available to following steps
             echo "export TAG=${TAG}" >> $BASH_ENV
   clair-scan:
-    description: "Vulnerability scan a Docker image"
+    description: "Clair: Vulnerability scan a Docker image"
```

Comment:
Is it worth keeping clair and trivy running?

(Let me guess: they find different things)

Reply:
correct.

Reply:
https://github.com/astronomer/issues/issues/743#issuecomment-591687980